### PR TITLE
Lowery_HW_1

### PR DESCRIPTION
**Pull request by DanielKLowery**

### CODE REVIEW

**Comment by DanielKLowery** [comment]
Here we go

**Review by SophiaMo — COMMENTED** [review]
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |           |  x            |            |
| **Achievement, creativity** |  x         |              |            |
| **Ease of access**          |  x         |              |            |

Daniel, I just added into the class yesterday and was not assigned to review, so I choose to review your work. I believe you did a great job with everything, including adding an image and links, and creating lists! I also like how you use bold and italic texts to summarize the process of getting changes into the README file. Your coding strategy is efficient as well. The table at the very top with your name, date, and output seems slightly distracting to me, so it might be a good idea to exclude it next time.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+* Game Theory
+
+Outside of research, my main hobbies are:
+
+* Hiking
+* Powerlifting
+* Pokemon Go
+
+Here is a picture of myself and Jack (my traveling duck) on a glacier
+![Jack](https://github.com/DanielKLowery/mycomputingrep/blob/master/IMG5981.JPG)
```
Comment (nnickels) [inline README.md:40]: Nice work with pic ... and great pic!

**Review by nnickels — COMMENTED** [review]
Evaluation HW-01 for Daniel Lowery
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X     |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |           |      X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Great job with the first homework! You use lots of markdown elements and your README renders clean and well.
* It's a tricky learning curve, but yes, it will definitely pay off! 
* For your table output you'll get in the hang of changing your output as well, such as github_document, etc.

**Pull request closed without merging**